speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/ResearcherMenu.cpp

```diff
@@ -327,6 +335,53 @@ static void OnActivate(void* /* dummy */)
     LoadDefaultSettings();
 }
 
+/// @brief Opens a file dialog and changes the button to reflect this choice.
+static void SelectFile(void* /* dummy */)
```

Comment:
Please format this function a bit better. Currently you have about 4 lines worth of code on one line. 
Some new lines may also help with readability.